modules/flight_interface/flight_interface.py

```diff
@@ -35,6 +38,13 @@ def create(cls, address: str, timeout_home: float) -> "tuple[bool, FlightInterfa
         # Get Pylance to stop complaining
         assert home_location is not None
 
+        result, flight_interface_logger = logger.Logger.create("flight_interface")
```

Comment:
If logger is created before the TCP port could we log issues with opening the port as well? In the past this has been an issue where we encountered errors eh?

Reply:
Good point. Created it first so that we can log the other errors in the create method.